aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AwsIotEndpointUtility.java

```diff
@@ -7,50 +7,29 @@
  * Helper class for parsing AWS IoT endpoints.
  */
 public final class AwsIotEndpointUtility {
```

Comment:
It's likely this helper class doesn't need to exist. It's also equally likely the client is doing this extraction unnecessarily.

Reply:
I moved the method from this class into the client itself. The extraction is necessary for IAM authentication, apparently.